static/about.html

```diff
@@ -239,11 +240,11 @@ <h2>
                 involved in CalyxOS have also been heavily involved in encouraging harassment
                 targeting GrapheneOS developers.</p>
 
-                <p>GrapheneOS is a hardened mobile OS with Android app compatibility focused on the 
```

Comment:
Can you drop these whitespaces changes please

Reply:
Sure, can do.